### DISCUSSION TITLE
New Robots

### DISCUSSION THREAD

**Discussion by DaniCoimbra** [NVlabs/curobo#286] [Software Q&A] [answered]
Hi,

I am interested in using the software with new robots, I saw the Tutorial section "Configuring a New Robot".
But on https://curobo.org/notes/01_robot_list.html "Supported Robots", there is a limited number of options.

Does that mean that if I try to configure an external robot, cuRobo wont support?

Thanks for the help!

**Comment by balakumar-s** [comment, marked as answer]
You can configure your robot following the tutorial and everything will work.

We might have to rename "Supported Robots" to "Pre-Configured Robots" to make it clear that we are providing configuration files for some robots.

**Reply by DaniCoimbra** [reply]
Thank you. I was able to add my robot.
